### PR TITLE
fix(@desktop/wallet): Edit Networks: Add warning when failover and main rpc are the same

### LINKED ISSUE #11551: Edit Networks: Add warning when failover and main rpc are the same
### Description

We can show a small warning message when the user edits the network to have main rpc url = failover rpc url

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
…in rpc are the same

closes #11551

### What does the PR do

Implements https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=18442-531470&mode=dev when Main and Failover RPCS are the same

### Affected areas

Profiel Wallet Networks Settings

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://github.com/status-im/status-desktop/assets/60327365/395bec13-76a3-46fc-bd2e-178354b79cc0

### CODE REVIEW

**Review by saledjenic — APPROVED** [review]

File: ui/app/AppLayouts/Profile/views/wallet/EditNetworkForm.qml [hunk 1]
```diff
@@ -61,6 +60,10 @@ ColumnLayout {
                 return  qsTr("RPC appears to be either offline or this is not a valid JSON RPC endpoint URL")
             case EditNetworkForm.Verified:
                 return qsTr("RPC successfully reached")
+            case EditNetworkForm.Verified:
```
Comment (caybro) [inline ui/app/AppLayouts/Profile/views/wallet/EditNetworkForm.qml:63]: Twice the same thing? :)

Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/Profile/views/wallet/EditNetworkForm.qml]: Fixed :D

File: ui/app/AppLayouts/Profile/views/wallet/EditNetworkForm.qml [hunk 2]
```diff
@@ -166,27 +181,36 @@ ColumnLayout {
...
+                                             d.evaluationStatusFallBackRpc === EditNetworkForm.SameAsOther ?
                                                  Text.AlignLeft: Text.AlignRight
         errorMessageCmp.visible: d.evaluationStatusFallBackRpc !== EditNetworkForm.UnTouched
         errorMessageCmp.text: d.getUrlStatusText(d.evaluationStatusFallBackRpc, text)
-        errorMessageCmp.color: d.evaluationStatusFallBackRpc === EditNetworkForm.Pending ?
-                                   Theme.palette.baseColor1:
-                                   d.evaluationStatusFallBackRpc === EditNetworkForm.Verified ?
-                                       Theme.palette.successColor1 : Theme.palette.dangerColor1
-
+        errorMessageCmp.color: {
```
Comment (caybro) [inline ui/app/AppLayouts/Profile/views/wallet/EditNetworkForm.qml:203]: These error colors are also repeated already, first around line 160

**Review by caybro — APPROVED** [review]
LGTM

**Pull request merged**